src/main/java/Duke.java

```diff
@@ -1,10 +1,440 @@
+            System.out.println("*");
+        }
+        for (int i = 0; i < maxDescriptionsLength + 2; i++) {
+            System.out.print("-");
+        }
+    }
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        //Prints all the welcome screens
+        printLogo();
+        printWelcomeMessage();
+        printVersionDescription();
+        System.out.println("\nLet's start:");
+        Task[] tasks = new Task[MAX_TASKS];
+        while (true) {
+            readCommand(tasks);
+        }
+    }
+
+    /**
+     * Reads the input command entered by the user and handles each command
+     *
+     * @param tasks The array to store all the tasks required
+     */
+    private static void readCommand(Task[] tasks) {
+        String command = in.nextLine().trim();
+        String[] words = command.split(" ");
+        if (isCommandEmpty(command)) {
+            System.out.println("\t(Empty) <- will not save to the list");
+        } else if (isCommandViewPersonality(command)) {
+            printPersonality();
+        } else if (isCommandExit(command)) {
+            showMessage("Bye! Hope to see you again :D");
+            System.exit(0);
+        } else {
+            if (isCommandList(command)) {
+                printToDoList(tasks, Task.totalTask, longestTaskDescription);
+            } else if (isCommandDone(words[0])) {
+                handleTaskDone(tasks, words);
+            } else if (isCommandAddTask(words[0])) {
+                addTask(tasks, command, words);
+            } else {
+                showMessage("Sorry, the command is invalid :(");
+            }
+        }
+    }
+
+    /**
+     * Checks if the command is meant to add task to the to-do list
+     *
+     * @param word The array of words that compose the input command
+     * @return Returns true if the command contains keywords "deadline", "event" or "todo"
+     */
+    private static boolean isCommandAddTask(String word) {
+        return word.equalsIgnoreCase("DEADLINE") || word.equalsIgnoreCase("EVENT") || word.equalsIgnoreCase("TODO");
+    }
+
+    /**
+     * Performs the add task action
+     *
+     * @param tasks   The array that stores all the tasks
+     * @param command The input command typed by the user
+     * @param words   The array of words that compose the input command
+     */
+    private static void addTask(Task[] tasks, String command, String[] words) {
+        if (isCorrectToDo(tasks, command, words) || isCorrectDeadline(tasks, command, words) || isCorrectEvent(tasks, command, words)) {
+            showMessage(" Class type [" + tasks[Task.totalTask].getClassType() + "] \"" + tasks[Task.totalTask] + "\" has been added to the list!"
```

Comment:
Can potentially put this into a function? (i.e. use a createMessage function)

Reply:
Thanks for your suggestion. I will consider that! :)